os/android/iahwc2.cpp

```diff
@@ -745,8 +745,10 @@ HWC2::Error IAHWC2::Hwc2Layer::SetLayerBuffer(buffer_handle_t buffer,
 
   native_handle_.handle_ = buffer;
   hwc_layer_.SetNativeHandle(&native_handle_);
-  if (acquire_fence > 0)
+  if (acquire_fence > 0) {
     hwc_layer_.SetAcquireFence(dup(acquire_fence));
+    close(acquire_fence);
```

Comment:
We could avoid dup and close here. Just pass the acquire_fence i.e  hwc_layer_.SetAcquireFence(acquire_fence);

This should be enough. I can revise this and merge

Reply:
Done. The patch is changed as below